obsei/preprocessor/text_splitter.py

```diff
@@ -0,0 +1,70 @@
+import traceback
+import logging
+from pydantic import Field
+from typing import Any, List, Optional, Tuple
+
+from obsei.payload import TextPayload
+from obsei.preprocessor.base_preprocessor import (
+    BaseTextPreprocessor,
+    BaseTextProcessorConfig,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class TextSplitterConfig(BaseTextProcessorConfig):
+    max_split_length: int = 512
+    split_stride: Optional[int] = 0  # overlap length
```

Comment:
What about having `generate_text_id` flag, So if this flag is True. Splitter will generate id to each incoming `input_list` and add that id in meta as let's say "text_id". It would useful incase later user aggregate split chunks.